virt/virt-4-11-release-notes.adoc

```diff
@@ -130,6 +128,10 @@ To preserve and support your existing nmstate configuration, install the xref:..
 
 //CNV-18323 VM templates can be made public i.e. available to all users
 
+* As of version 4.11, `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone operator through the Operator Lifecycle Manager (OLM) in the OperatorHub. 
++
+If you are using OpenShift Virtualization 4.10 or earlier, you must move all nodes out of maintenance mode prior to upgrading to OpenShift 4.11. If a node must stay in maintenance mode during the upgrade, the cluster admin must install the standalone NMO and replace the `nodemaintenances.nodemaintenance.kubevirt.io` CR with a nodemaintenances.nodemaintenance.medik8s.io CR before upgrading to OpenShift Virtualization 4.11. You will receive an error and be unable to upgrade if you attempt to upgrade with a `nodemaintenances.nodemaintenance.kubevirt.io` CR.
```

Comment:
`4.10 or earlier, you must move all nodes out of maintenance mode prior to upgrading to OpenShift 4.11` - this is misleading. 4.10.2 onwards upgrade is blocked, so prior versions would have no issues. Also, upgrading to ocp 4.11 is not blocked. if you have upgraded ocp to 4.11 and you are on >= 4.10.2 cnv, only then you must move nodes out of maintenance or install standalone NMO.

Reply:
@dbasunag Would it be accurate then to replace "4.10 or earlier" with "4.10.1 or earlier"?

Reply:
It should be 4.10.2 or later. Earlier cnv versions won't block upgrade due to NMO.

Reply:
Also user does not need to do anything prior to upgrading to ocp 4.11. This would only impact cnv upgrade `after` ocp has been upgraded to 4.11.